modules/accredited_representative_portal/config/routes.rb

```diff
@@ -22,6 +22,8 @@
     post '/submit_representative_form', to: 'representative_form_upload#submit'
     post '/representative_form_upload', to: 'representative_form_upload#upload_scanned_form'
 
+    resources :submissions, only: :index
```

Comment:
Let's solidify the name of this resource that our API exposes as `/claim_submissions`.

Reply:
One day we may get to consolidate the write paths visible above this in a Rails-resourceful way too!

Reply:
In fact this idea of resourcefulness is one of the absolute top ideas I tend to keep in mind for all typical web app development.